lib/gradient/ast_specifier.ex

```diff
@@ -625,6 +589,29 @@ defmodule Gradient.AstSpecifier do
     end
   end
 
+  @doc """
+    Update location in call args with the support to the pipe operator.
```

Comment:
As the `@doc` follows the markdown syntax space has a meaning
https://stackoverflow.com/a/40023279 
```suggestion
  Update location in call args with the support to the pipe operator.
```